mlir/tools/mlir-miopen-lib/CMakeLists.txt

```diff
@@ -97,4 +90,56 @@ if( BUILD_FAT_LIBMLIRMIOPEN )
   install(TARGETS MLIRMIOpen
     ARCHIVE DESTINATION lib)
 
+  # Install Miir.h to ${CMAKE_INSTALL_PREFIX}/include/<project-name>-<version>/
+  # as part of component libMLIRMIOpen
+  set(PACKAGE_DIR MLIRMIOpen-${CMAKE_PROJECT_VERSION})
+  install(FILES Miir.h
+    DESTINATION include/${PACKAGE_DIR}
```

Comment:
Let's keep the `TYPE` in here?

Reply:
Ok, agreed on the DESTINATION bit. I do think we shouldn't put the version in `PACKAGE_DIR` though. (and see https://cmake.org/cmake/help/latest/command/install.html and the mention of `CMAKE_INSTALL_INCLUDEDIR`